### PR TITLE
Implemenent an easier method for client comms

### PR DESCRIPTION
**Pull request by ahmed-okkio**
https://trello.com/c/aRsPxJgm/96-clientcomms-class

### CODE REVIEW

File: Backend/Models/Messages/CommandMessage.cs [hunk 1]
```diff
@@ -4,61 +4,63 @@
 
 namespace Backend.Models.Messages
 {
-    enum CommandType
-    {
-        Mute,
-        Kick,
-        Ban,
-        Ipban,
```
Comment (ahmed-okkio) [inline Backend/Models/Messages/CommandMessage.cs:12]: Moved this down into the class

File: Backend/Models/Messages/CommandMessage.cs [hunk 2]
```diff
@@ -4,61 +4,63 @@
...
-        Unban,
-        Unbanip,
-        Unknown
-    }
+
```
Comment (ahmed-okkio) [inline Backend/Models/Messages/CommandMessage.cs:7]: Moved the enum down into the class

File: Backend/Models/Messages/CommandMessage.cs [hunk 3]
```diff
@@ -4,61 +4,63 @@
...
         private string? _payload;
 
+        public CommandMessage(WebSocket socket, string rawString) : base(socket, rawString)
+        {
+            _command = GetCommandType();
+            _sender = UserManager.GetUserBySocket(socket);
+            _target = _to;
+            _channel = _in;
+            _payload = _with;
+        }
```
Comment (ahmed-okkio) [inline Backend/Models/Messages/CommandMessage.cs:35]: Moved the constructor to the top of the class

File: Backend/Models/Messages/CommandMessage.cs [hunk 4]
```diff
@@ -4,61 +4,63 @@
...
+                case "ipban":
+                    return CommandType.Ipban;
+                case "unban":
+                    return CommandType.Unban;
+                case "unbanip":
+                    return CommandType.Unbanip;
+                default:
+                    return CommandType.Unknown;
+            }
+        }
```
Comment (ahmed-okkio) [inline Backend/Models/Messages/CommandMessage.cs:56]: Moved GetCommandType to the top of the class

File: Backend/ServerModules/LSMPBehavior.cs [hunk 5]
```diff
@@ -1,9 +1,40 @@
 ﻿
+using Backend.Models.Messages;
+using Backend.Models.Users;
 using WebSocketSharp.Server;
 
 namespace Backend.ServerModules
 {
-    public class LSMPBehavior:WebSocketBehavior
+    public class LSMPBehavior: WebSocketBehavior
```
Comment (OmarQurashi868) [inline Backend/ServerModules/LSMPBehavior.cs]: what zis?

**Review by OmarQurashi868 — APPROVED** [review]
lgtm? 👀

**Pull request merged**